spec/readthis/passthrough_spec.rb

```diff
@@ -1,15 +1,18 @@
 RSpec.describe Readthis::Passthrough do
+  let(:value) { 'skywalker' }
   describe '.load' do
     it 'passes through the provided value' do
-      value = Object.new
       expect(Readthis::Passthrough.load(value)).to eq(value)
     end
   end
 
   describe '.dump' do
     it 'passes through the provided value' do
-      value = Object.new
       expect(Readthis::Passthrough.dump(value)).to eq(value)
     end
+
+    it 'returns a duplicate of passed value' do
+      expect(Readthis::Passthrough.dump(value)).not_to be(value)
```

Comment:
This can be put inside the previous `it` block, both expectations describe the pass through behavior.